utils/python/tests/scripts_regression_tests.py

```diff
@@ -1192,92 +1229,6 @@ def test_save_timings_manual(self):
         self.simple_test(manual_timing=True)
 
 ###############################################################################
-class C_TestXMLQuery(unittest.TestCase):
```

Comment:
They were not deleted, they were moved into TestCreateNewcase (and improved upon)